src/main/java/edu/tamu/scholars/middleware/export/service/CsvExporter.java

```diff
@@ -121,11 +121,20 @@ private List<Object> getRow(
             if (content.containsKey(property)) {
 
                 Object value = content.get(property);
+                if (property.equals("type")) {
+
+                    List<String> values = asStringList(value);
```

Comment:
```suggestion
                    List<String> values = asStringList(value);
```

I suggest this style clean up by removing the extra line.